starport/pkg/cosmosaccount/cosmosaccount.go

```diff
@@ -0,0 +1,216 @@
+package cosmosaccount
+
+import (
+	"errors"
+	"os"
+
+	dkeyring "github.com/99designs/keyring"
+	"github.com/cosmos/cosmos-sdk/crypto/hd"
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/go-bip39"
+)
+
+// KeyringServiceName used for the name of keyring in OS backend.
+const KeyringServiceName = "starport"
+
+// KeyringHome used to store account related data.
+var KeyringHome = os.ExpandEnv("$HOME/.starport/accounts")
+
+var (
+	ErrAccountExists       = errors.New("account already exists")
+	ErrAccountDoesNotExist = errors.New("account does not exist")
+)
+
+const (
+	accountPrefixCosmos = "cosmos"
+	pubKeyPrefix        = "pub"
```

Comment:
The prefix used for public key is actually `accountPrefix + "pub"`

It could even be theorically fully personalized although for Starport scaffolded chains we don't let this capability in `spm` and always use prefix + "pub": https://github.com/tendermint/spm/blob/e8ed7e137572290341e8fb716e53f8d05f9b718d/cosmoscmd/prefix.go#L15

Reply:
I agree but not subject of this pr. 👍

Reply:
Well I may think the only change to do is to have
```
conf.SetBech32PrefixForAccount(accPrefix, accPrefix+pubKeyPrefix)
```
at line 65 and 81

Reply:
2fffc754